extra-files/cloud_setup/bash_history

```diff
@@ -0,0 +1,3 @@
+sudo su galaxy
```

Comment:
This file is totally a personal preference and it's just there because one of these commands is typically what I type right after I log in. Do you prefer to omit it from the repo?